### PR TITLE
polyfill symbol in BlackBerry

### PR DESCRIPTION
**Pull request by wheresrhys**
@JakeChampion being deliberately conservative with versions given the compatibility problems we've had with symbol before. FT only needs v10 covered. If others want to invest time in verifying Symbol and related polyfills don't blow up in BB<10 then config can be modified in future PRs

### CODE REVIEW

**Comment by JakeChampion** [comment]
@wheresrhys Do you have screenshots of the tests passing on BB10?

**Comment by wheresrhys** [comment]
not of the tests, but ft.com is fixed by forcing `Array.from` and `Map` to be bundled on BB10, which is reasonable evidence that this is ok. I'll work on running the tests this afternoon though

File: polyfills/Array/from/config.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
 		"firefox_mob": "<32",
 		"android": "*",
 		"ios_saf": "<9",
-		"samsung_mob": "*"
+		"samsung_mob": "*",
+		"bb": ">=10"
```
Comment (JakeChampion) [inline polyfills/Array/from/config.json]: I think this needs to be `10 - *` to be valid semver.

**Review by JakeChampion — CHANGES_REQUESTED** [review]

**Comment by wheresrhys** [comment]
image

**Review by JakeChampion — APPROVED** [review]

**Comment by JakeChampion** [comment]
@wheresrhys This is now live on http://qa.polyfill.io - would you be able to report back whether we have fixed your issue?

**Pull request merged**